roles/rabbitmq/tasks/main.yml

```diff
@@ -1,8 +1,13 @@
 ---
+- name: Enable RDO repository
+  yum:
+    name: centos-release-openstack-mitaka
+  when: manage_repos|default(false)
```

Comment:
Since this repository isn't specific to rabbitmq, I wonder if this should be moved into a separate role? (e.g., `roles/rdo-mitaka`).  And what if someone wants a release other than mitaka?  Maybe parameterize with `rdo_release: mitaka` or something.